src/CiviDistManagerBundle/Controller/CheckController.php

```diff
@@ -180,6 +180,8 @@ protected function getLatestStable() {
           self::STABLE_DOWNLOAD_URL, $rev, $rev),
         'Joomla' => sprintf('%s/%s/civicrm-%s-joomla.zip',
           self::STABLE_DOWNLOAD_URL, $rev, $rev),
+        'Joomla5' => sprintf('%s/%s/civicrm-%s-joomla5.zip',
```

Comment:
With the changes in https://github.com/civicrm/civicrm-core/pull/31325 this become ... '-joomla5bc.zip'